web/packages/shared/utils/createMfaOptions.ts

```diff
@@ -16,31 +16,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Auth2faType, PreferredMfaType } from 'shared/services/types';
-
-export default function createMfaOptions(opts: Options) {
-  const { auth2faType, required = false } = opts;
-  const mfaOptions: MfaOption[] = [];
-
-  if (auth2faType === 'off' || !auth2faType) {
-    return mfaOptions;
-  }
-
-  const mfaEnabled = auth2faType === 'on' || auth2faType === 'optional';
+import { getMfaRegisterOptions } from 'teleport/services/mfa';
 
-  if (auth2faType === 'webauthn' || mfaEnabled) {
-    mfaOptions.push({ value: 'webauthn', label: 'Passkey or Security Key' });
-  }
-
-  if (auth2faType === 'otp' || mfaEnabled) {
-    mfaOptions.push({ value: 'otp', label: 'Authenticator App' });
-  }
-
-  if (!required && auth2faType === 'optional') {
-    mfaOptions.push({ value: 'optional', label: 'None' });
-  }
+import { Auth2faType, PreferredMfaType } from 'shared/services/types';
 
-  return mfaOptions;
+// Deprecated: use getMfaRegisterOptions or getMfaChallengeOptions instead.
+// TODO(Joerger): Delete once no longer used.
+export default function createMfaOptions(opts: Options): MfaOption[] {
+  return getMfaRegisterOptions(opts.auth2faType).map(opt => {
+    if (opt.value === 'totp') {
+      return {
+        value: 'otp',
+        label: opt.label,
+      };
+    }
+
+    return opt as MfaOption;
```

Comment:
Isn't `opt` already `MfaOption` here?

Reply:
There's `utils/MfaOption` and there is `services/mfa/MfaOption`. I'm not sure how to how to make the former a type alias of the latter since they have the same name, so I just did a type cast. Is there another way to make this work?

Reply:
would using `satisfies MfaOption` work here to coerce it w/out asserting?

Reply:
> would using satisfies MfaOption work here to coerce it w/out asserting?
Maybe? But I'm not sure how to import `services/mfa/MfaOption` since the name overlaps.

Reply:
`import { MfaOption as SomeOtherMfaOption }`